DailyGoals/DailyGoals/Entities/Goal.swift

```diff
@@ -9,9 +9,35 @@ import Foundation
 
 struct Goal: Identifiable, Hashable {
 
+	enum Action: CaseIterable, CustomStringConvertible {
+		case unknown
+		case devComplete
+		case readyForQA
+		case QAcomplete
+		case done
+		case additional
+
+		var description: String {
+			switch self {
+			case .unknown:
```

Comment:
do we really need unknown

Reply:
It's the default option when you create a new goal.
It could be nil, but that doesn't change the fact that we need to handle an unchosen value down the line, so I thought unknown would have make more sense.